src/components/Page.tsx

```diff
@@ -80,7 +100,11 @@ export function Page(props: {
     <View style={styles.container}>
       <View style={styles.titlePane}>
         <Text style={[styles.title, {color: colors.text}]}>{props.title}</Text>
-        {DisplayBadge(props.wrappedNativeControl)}
+        <View>
+          {DisplayNativeControlBadge(props.wrappedNativeControl)}
+          {DisplayCoreComponentBadge(props.coreComponent)}
+          {DisplayCommunityModuleBadge(props.communityModule)}
```

Comment:
isn't a component either Core or Community module? shouldn't we only have one badge that shows one thing or the other?

Reply:
Sure I can make that logic change!